app/consumer-democracy/app.go

```diff
@@ -149,6 +162,7 @@ var (
 		evidence.AppModuleBasic{},
 		transfer.AppModuleBasic{},
 		vesting.AppModuleBasic{},
+		tendermint.AppModuleBasic{},
 		// router.AppModuleBasic{},
 		consumer.AppModuleBasic{},
```

Comment:
Might want to add `x/consensus` see https://github.com/cosmos/cosmos-sdk/pull/12905

```suggestion
		consumer.AppModuleBasic{},
		consensus.AppModuleBasic{},
```

Reply:
Done